src/services/style-service.ts

```diff
@@ -66,8 +66,19 @@ enum ThemeAttribute {
 
 const resolveFontSize = (fontSize: string | undefined) => (fontSize ? `${parseInt(fontSize, 10)}px` : undefined);
 
-const resolveColor = (theme: ExtendedTheme, color: PaletteColor | undefined) =>
-  color ? theme.getColorPickerColor(color) : undefined;
+export const resolveColor = (theme: ExtendedTheme, color: PaletteColor | undefined) => {
+  if (color) {
+    const resolvedColor = theme.getColorPickerColor(color);
+    // Handle when color is set to "none" via the color picker
+    if (resolvedColor === "none") {
+      return Colors.Transparent;
```

Comment:
Just a question out of curiosity, why not returning undefined here? is it any different than returning transparent?

Reply:
Color is resolved in a specific order like this `resolveColorFromStylingPanel ?? colorFromTheme ?? fallbackColor`. So returning `undefined` would result in it using theme color instead.